web/docs/setup/tune.md

```diff
@@ -101,6 +101,23 @@ VAST supports [rebuilding the entire database](#rebuild-partitions) in case you
 want to switch to a different store format. However, VAST works perfectly fine
 with a mixed-storage configuration, so a full rebuild is not required.
 
+### Compression
+
+Partitions in feather and parquet format are compressed using zstd. To fine-tune
+the space-time trade-off, VAST allows you to control the compression level used:
```

Comment:
Please use active-voice, and start with the problem: As a user, why would I want to change the compression level?

Spelling-wise, Zstd, Feather, and Parquet are usually all capitalized.